extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -384,6 +384,13 @@ public void configurationDescriptorBuilding(
             return;
         }
 
+        if (shouldIgnorePersistenceXmlResources(hibernateOrmConfig)) {
+            LOG.warnf("Hibernate ORM ignores 'persistence.xml' files because config property '%s' or System property '%s' was set to true",
+                    "quarkus.hibernate-orm.persistence-xml.ignore",
+                    "SKIP_PARSE_PERSISTENCE_XML");
+            return;
```

Comment:
@Sanne Does not have to be a warning. 

The intention behind it was to warn (inform?) users that the 'persistence.xml' is not taken into an account. But as the system property is not documented and config prop should be in one place, it might not be any case for causing some confusion. You're right.

@Sanne As the flag is added intentionally, do you prefer to use INFO or just DEBUG?

Reply:
+1 we don't want a warning here.

Reply:
You can add a debugf if you wish but I would leave it at that.

Also, is `SKIP_PARSE_PERSISTENCE_XML` supposed to be exposed to the public? It looks like something internal.

Reply:
Not a proper solution. Please check the conversation below. Thanks.

Reply:
> Not a proper solution.

Hi, may I suggest when you say something like that to specify what you're referring to - there's a lot of ambiguity otherwise ;)

But as we said, we won't log this as a warning. Since you have a custom extension (yes I did read the conversation below) you could of course log a warning in your own extension.

HTH